hack/app_sre_create_image_catalog.sh

```diff
@@ -0,0 +1,111 @@
+#!/bin/bash
+
+set -exv
+
+BRANCH_CHANNEL="$1"
+QUAY_IMAGE="$2"
+
+GIT_HASH=$(git rev-parse --short=7 HEAD)
+GIT_COMMIT_COUNT=$(git rev-list 9a5e025f954009144cdc403acbdc3cb5178addf5..HEAD --count)
```

Comment:
in the interest of being able to reuse this, I would replace this with:

    GIT_COMMIT_COUNT=$(git rev-list $(git rev-list --max-parents=0 HEAD)..HEAD --count)

Reply:
Nice one! I have to replicate this to 3 other repos so I plan to create a repo or something to centralize this and make it reusable.